### PR TITLE
Add some Helpful Chinese Docs

### PR DESCRIPTION
**Pull request by BlueSkyXN**

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## 认证和各项准备工作
+
+在使用之前，你需要完成一系列准备工作
+
+1. 准备ChatGPT账号，最好的PLUS订阅的，有没有开API不重要
+2. 完善的运行环境和网络环境（否则你总是要寻找方法绕过）
+3. Access Token和PUID，下面会教你怎么获取
+4. 选择一个代理后端或者自行搭建
+5. 你可以在 https://github.com/BlueSkyXN/OpenAI-Quick-DEV 项目找到一些常用组件以及一些快速运行的教程或程序。
```
Comment (acheong08) [inline README_CN.md:16]: 👀

File: cases/sider/Sider-Config.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# 这是一个指导你如何在Sider程序中应用本程序的API
+Sider ，一个浏览器插件
+
+https://chrome.google.com/webstore/detail/difoiogjjojoaoomphldepapgpbgkhkb
```
Comment (acheong08) [inline cases/sider/Sider-Config.md:4]: Is this open source?

Comment (BlueSkyXN) [reply cases/sider/Sider-Config.md:4]: I am no sure. 
It is a guide to USE ChatGPT-to-API to Crack Any OPENAI API KEY Setting.
I have nothing to do with them, I just do a show to be used on other platforms

File: README_CN.md (hunk 1 above)
Comment (acheong08) [reply README_CN.md:16]: Reviewed the code. No malware. Will accept this

**Pull request merged**